apps/web/app/constants.ts

```diff
@@ -194,3 +194,71 @@ export enum IssuesView {
 export const TaskStatus = {
 	INPROGRESS: 'in-progress'
 };
+
+export const languagesFlags = [
+	{
+		flag: 'https://twemoji.maxcdn.com/2/svg/1f1ec-1f1e7.svg',
```

Comment:
We will eventually translate to MANY languages, so almost all "flag" of country will be needed later and also I am pretty sure it will NOT load those images that not used in frontend! So let's search for good package and as I asked - create a list of such packages and let me know which one you suggest to use. You can also copy / paste SVGs, like we do in some other projects etc. Just NOT reference some public URLs. But package is really preferred way to do it I think. I.e. some well known package with SVGs for "Flags" :)